src/dropdown.ts

```diff
@@ -299,55 +308,54 @@ export default class Dropdown extends LitElement implements FormControl {
   }
 
   set value(value: string[]) {
-    // You'll notice `this.#value` is internally often used directly. That's done to
-    // bypass the setter's side effects to either avoid a bad state or a prevent a loop.
-    //
-    // If `#onOptionsSelectedChange()`, for example, weren't to bypass the setter, then
-    // our setting value below would cause `#onOptionsSelectedChange()` to be called
-    // again, which in turn would call this setter.
```

Comment:
Moved the gist of this comment to [`#isSelectionFromValueSetter`](https://github.com/CrowdStrike/glide-core/pull/932/files#diff-7431ab16692f7d7a7cfecdb04f55d6c62f87417591065650d88dc407a9b62f3bR1341), which is needed regardless of whether `this.value` is set directly.